Cargo.toml

```diff
@@ -33,10 +33,11 @@ tokio-compat-02 = "0.2.0"
 tokio-stream = "0.1.7"
 url = "2.2.2"
 xdg = "2.2"
-librespot-audio = { version = "0.2.0", default-features=false, features = ["with-tremor"] }
-librespot-playback = { version = "0.2.0", default-features=false }
-librespot-core = { version = "0.2.0"}
-librespot-connect = { version = "0.2.0"}
+librespot-audio = { version = "0.4.0", default-features = false }
+librespot-playback = { version = "0.4.0", default-features = false }
+librespot-core = { version = "0.4.0" }
+librespot-discovery = { version = "0.4.0" }
+librespot-connect = { version = "0.4.0" }
```

Comment:
The most recent version of `librespot` seems to be `0.4.2`. Should we just omit the last number? Or have there been changes between `0.4.0` and `0.4.2`, such that this doesn't yet work with `0.4.2`?

Reply:
I just omitted the last number, in accordance to #1183